test/test.ts

```diff
@@ -854,58 +847,60 @@ describe('gcs-resumable-upload', () => {
     it('should set userProject', (done) => {
       up.authClient = {
         authorizeRequest(reqOpts: RequestOptions) {
-          assert.deepEqual(reqOpts.qs, {userProject: USER_PROJECT});
+          assert.deepEqual(reqOpts.params, {userProject: USER_PROJECT});
           done();
         }
       };
       up.getRequestStream(REQ_OPTS);
     });
 
     it('should destroy the stream if an error occurred', (done) => {
-      const error = new Error(':(');
+      const error = 'ಠ_ಠ';
```

Comment:
nit: a better variable name would be `message` or `MESSAGE`, or even `LOOK_OF_DISAPPROVAL`.

Reply:
🤣 but why